vectorizers/preprocessing.py

```diff
@@ -36,6 +37,33 @@ def construct_document_frequency(token_by_doc_sequence, token_dictionary):
     return doc_freq / len(token_by_doc_sequence)
 
 
+def construct_timed_document_frequency(token_by_doc_sequence, token_dictionary):
+    """Returns the frequency of documents that each token appears in.
+
+    Parameters
+    ----------
+    token_by_doc_sequence: Iterable
+        A sequence of sequences of tokens
```

Comment:
I think this should be "a sequence of sequences of (token, timestamp) pairs." Or something like that. Correct?